packages/airnode-examples/package.json

```diff
@@ -15,6 +15,7 @@
     "create-airnode-config": "ts-node src/scripts/create-airnode-config.ts",
     "create-airnode-secrets": "ts-node src/scripts/create-airnode-secrets.ts",
     "create-aws-secrets": "ts-node src/scripts/create-aws-secrets.ts",
+    "deploy-authorizers": "ts-node src/scripts/deploy-authorizers.ts",
```

Comment:
This is specific to the examples-e2e so it should be created inside the file of that integration. The `scripts` should be as generic as possible otherwise it will get messy.

Reply:
Good call- I moved it.